src/go/rpk/pkg/cli/cmd/cluster/config/import.go

```diff
@@ -119,13 +128,16 @@ func importConfig(
 				upsert[k] = v
 			}
 		} else {
-			// Present in input but not original config, insert
-			upsert[k] = v
-			propertyDeltas = append(propertyDeltas, propertyDelta{k, "", fmt.Sprintf("%v", v)})
+			// Present in input but not original config, insert if it differs
+			// from the materialized current value (which may be a default)
+			if !haveOldValMaterialized || !reflect.DeepEqual(oldValMaterialized, v) {
+				upsert[k] = v
+				propertyDeltas = append(propertyDeltas, propertyDelta{k, fmt.Sprintf("%v", oldValMaterialized), fmt.Sprintf("%v", v)})
+			}
```

Comment:
Summary of this block:
* If this old value did not exist in the _full configuration_, this is an upsert. @jcsp is this even valid? What's the use in adding a configuration value if it is not supported (doesn't even have a default)? However, this block of code is not a PR blocker because this same condition existed in the old code.
* If the value is different from the old default, this is a change from a default value and is an upsert.